From a09e9aa47dcaf9dccd798dba57a049ab829b40db Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Thu, 8 Jul 2021 22:50:55 -0400
Subject: [PATCH 14/18] Drop dependency on pymongo

Can lead to a circular dependency. Plus, we don't run the entire test
suite in this recipe anyway.
---
 etc/pip/components/core.req | 1 -
 1 file changed, 1 deletion(-)

diff --git a/etc/pip/components/core.req b/etc/pip/components/core.req
index f888c6f65e..4692581447 100644
--- a/etc/pip/components/core.req
+++ b/etc/pip/components/core.req
@@ -1,6 +1,5 @@
 # Core (we need these for most buildscripts)
 psutil <= 5.8.0
-pymongo >= 3.9, < 4.0
 PyYAML >= 3.0.0, <= 6.0.0
 requests >= 2.0.0, <= 2.26.0
 pkce == 1.0.3
-- 
2.34.1

